### PR TITLE
rename Master output to Main output in user visible strings

### PR DESCRIPTION
**Pull request by Be-ing**
There is still more work to remove this terminology from the code.
"Master" is still used in Shade but I do not know how to change
that because it is in a raster graphic in
res/skins/Shade/style/style_bg_mixer_panel_main.png and I do not
know what font to use to replace it.

### CODE REVIEW

**Comment by uklotzde** [comment]
This PR requires us to agree on an exception from *string freeze* for beta releases.

No dissent, just a formal note for the record.

**Comment by daschuer** [comment]
Let's retarget this to master.
We have no benefit if this leads to many untranslated strings in our release.

**Comment by Be-ing** [comment]
I'm okay with either.

**Comment by daschuer** [comment]
The Schade lable is hand drawn.

File: src/preferences/dialog/dlgprefsounddlg.ui [hunk 1]
```diff
@@ -148,7 +148,7 @@
      <item row="11" column="0">
       <widget class="QLabel" name="masterDelayLabel">
        <property name="text">
-        <string>Master Delay</string>
+        <string>Main Output Delay</string>
```
Comment (daschuer) [inline src/preferences/dialog/dlgprefsounddlg.ui:151]: We should either omit Output here or add Output also to Headphone and Booth Delay

File: src/soundio/soundmanagerutil.cpp [hunk 2]
```diff
@@ -154,7 +154,7 @@ QString AudioPath::getStringFromType(AudioPathType type) {
         // handle this -- bkgood
         return QStringLiteral("Invalid");
     case MASTER:
-        return QStringLiteral("Master");
+        return QStringLiteral("Main");
```
Comment (daschuer) [inline src/soundio/soundmanagerutil.cpp:157]: Since this is the programmatic name, I am afraid this likely breaks the user config.

**Review by daschuer — CHANGES_REQUESTED** [review]

File: src/preferences/dialog/dlgprefsounddlg.ui (hunk 1 above)
Comment (Be-ing) [reply src/preferences/dialog/dlgprefsounddlg.ui:151]: I think "Main Delay" by itself would be a bit unclear what it is referring to.

Comment (Be-ing) [reply src/preferences/dialog/dlgprefsounddlg.ui:151]: I'll add "Output" to the rest of the delay labels for consistency.

File: src/soundio/soundmanagerutil.cpp (hunk 2 above)
Comment (Be-ing) [reply src/soundio/soundmanagerutil.cpp:157]: okay I see the comment above the function explaining that now

**Comment by Be-ing** [comment]
Ready for merge?

**Comment by daschuer** [comment]
Can you take care of the Shade label as well? Otherwise I can issue a PR to your branch once I find time.

**Comment by Be-ing** [comment]
> Can you take care of the Shade label as well? Otherwise I can issue a PR to your branch once I find time.

I don't know how to. I have no idea what font that is.

**Comment by ronso0** [comment]
this is no special font.
you can simply copy the lettets from other label images.

**Comment by Be-ing** [comment]
Really? @daschuer can you confirm that's the only way?

**Comment by daschuer** [comment]
Confirmed. These are hand written fonts, pixel by pixel.

**Comment by Be-ing** [comment]
Okay I'll leave that for you to do.

**Comment by daschuer** [comment]
https://github.com/Be-ing/mixxx/pull/48

**Comment by daschuer** [comment]
Thank you. LGTM.

**Pull request merged**